services/thumbnails/pkg/service/grpc/v0/decorators/tracing.go

```diff
@@ -27,7 +27,10 @@ func (t tracing) GetThumbnail(ctx context.Context, req *thumbnailssvc.GetThumbna
 
 	if thumbnailsTracing.TraceProvider != nil {
 		tracer := thumbnailsTracing.TraceProvider.Tracer("thumbnails")
-		ctx, span = tracer.Start(ctx, "Thumbnails.GetThumbnail")
+		spanOpts := []trace.SpanStartOption{
+			trace.WithSpanKind(trace.SpanKindServer),
+		}
+		ctx, span = tracer.Start(ctx, "Thumbnails.GetThumbnail", spanOpts...)
```

Comment:
hm this will only set the span kind for thumbnails 🤔

Reply:
I've updated with the other missing ones. I hope I didn't miss any.